src/Application.php

```diff
@@ -46,14 +46,25 @@ public function get()
     {
         $this->setParameters();
 
+        $this->restURI = $this->getHost() . "/servers/" . $this->getServerInstance() . "/vhosts/" . $this->getVHostInstance() . "/applications/{$this->name}";
+
+        return $this->sendRequest($this->preparePropertiesForRequest(self::class), [], self::VERB_GET);
+    }
+
+    public function getAdvanced()
+    {
+        $this->setParameters();
+
+        $this->restURI = $this->getHost() . "/servers/" . $this->getServerInstance() . "/vhosts/" . $this->getVHostInstance() . "/applications/{$this->name}/adv";
+
         return $this->sendRequest($this->preparePropertiesForRequest(self::class), [], self::VERB_GET);
     }
 
     public function getAll()
     {
         $this->setParameters();
 
-        $this->restURI = $this->getHost() . '/servers/' . $this->getServerInstance() . '/vhosts/' . $this->getVHostInstance() . '/applications';
+        $this->restURI = $this->getHost() . "/servers/" . $this->getServerInstance() . "/vhosts/" . $this->getVHostInstance() . "/applications";
```

Comment:
i think single quote usage is more better :) 
also it would be good if you use a private method such as `getRestUri()`. As i can see, you are generating restUri almost in every method.

Reply:
I myself prefer single quotes, but double quotes were already been used inside the file and I was trying to follow the same coding style.

As for the `restUri` the reason I generate it in every method, is because when one method gets called it may change the `restUri` that is being set by the constructor, and then as a result if you call another method later it's going to use the wrong URI. Take a look [here](https://github.com/WowzaMediaSystems/wse-rest-library-php/blob/master/src/Application.php) for example, if you call the `getAll` method and after that you call `get` it is going to use the wrong URI.